server/channels/api4/system.go

```diff
@@ -414,6 +415,29 @@ func getLogs(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.ArrayToJSON(lines)))
 }
 
+func downloadLogs(c *Context, w http.ResponseWriter, r *http.Request) {
+	lines, _ := c.App.GetLogs(c.AppContext, c.Params.Page, c.Params.LogsPerPage)
```

Comment:
Huh, I assumed it was something more complicated than just reading directly from the log file because I wouldn't expect that to work in a multi-server environment. I guess not.

In that case, take a look at the code linked by @pvev to find how to get the log file location, but instead of using `os.ReadFile` to read the file into memory as an array of bytes, you can probably use `os.Open` to get a reader which can be passed directly to `WriteFileResponse`